cmd/fyneterm/theme.go

```diff
@@ -23,9 +23,15 @@ func (t *termTheme) Color(n fyne.ThemeColorName, v fyne.ThemeVariant) color.Colo
 			return c
 		}
 		if v == theme.VariantLight {
-			return color.NRGBA{R: 0xaa, G: 0xaa, B: 0xaa, A: 0xf6}
+			return color.NRGBA{R: 85, G: 85, B: 255, A: 0xf6}
```

Comment:
This cannot be right? it replaces what was a light grey with a vivid blue color...

Reply:
I picked a color appropriate for a light-mode terminal. I felt it was subjective. maybe we should put all this into a config file?

Reply:
What was wrong with the light grey background color? in light node most apps have a nearly-white background with dark text.

Reply:
ok this has been resolved